src/site/_data/lib/hash.js

```diff
@@ -52,9 +62,9 @@ const hashForProdCache = {};
  * @return {string}
  */
 function hashForProd(file) {
-  if (!isProduction) {
+  /*if (!isProduction) {
     return `${file}?v=${randomHash()}`;
-  }
+  }*/
```

Comment:
If this isn't commented out, there would be a unique script hash for every single page. This would still technically work, but it would make the CSP unnecessarily long.